core/domain/exp_domain.py

```diff
@@ -2968,43 +2968,62 @@ def delete_state(self, state_name):
         del self.states[state_name]
 
 
-    def get_trainable_states_dict(self, old_states, change_list):
+    def get_state_names_mapping(self, change_list):
+        """Creates a mapping from the current state names of the exploration
+        to their corresponding state names of the older version.
+
+        Args:
+            change_list: list(dict). A list of changes introduced in this
+                commit.
+
+        Returns:
+            dict. The state_names_mapping dict.
+        """
+        state_names_mapping = {}
+        renamed_cmds = []
+        for change_dict in change_list:
+            if change_dict['cmd'] == CMD_RENAME_STATE:
+                renamed_cmds.append(change_dict)
+
+        for state_name in self.states:
+            state = self.states[state_name]
+            if not state.can_undergo_classification():
```

Comment:
This check has nothing to do with the point of this function; remove it please. We could conceivably use this mapping for other things in the future.